src/GitInfo.php

```diff
@@ -8,12 +8,79 @@ class GitInfo
      * @var
      */
     protected $path;
+    protected static $registeredCommands = [
+        'latest-commit' => 'git log --format="Revision: %H%nAuthor: %an (%ae)%nDate: %aI%nSubject: %s" -n 1',
+        'all-tags' => 'git tag'
+    ];
 
-    public function __construct($path = null)
+    /**
+     * GitInfo constructor.
+     * @param string $path
+     * @param array $commands
+     */
+    public function __construct($path = __DIR__, array $commands = [])
```

Comment:
`__DIR__` is useless here. Let's set `$path`'s default to `null` and set `$this-path` to `getcwd()` when `$path` is empty:
```suggestion
    public function __construct($path = null, array $commands = [])
```